model/sklearn/sklearn.py

```diff
@@ -25,36 +25,23 @@ def __init__(self, id: str, config: SKLearnConfig):
     def preload(self):
         self.nlp = get_spacy()
         self.spacy_stopwords = spacy.lang.en.stop_words.STOP_WORDS
-        self.preprocess = create_preprocess(self.nlp)
+        # self.preprocess = create_preprocess(self.nlp)
 
-    def fit(self, dataset: pd.DataFrame) -> None:
-
-        X_train = dataset[Const.input_col].swifter.apply(self.preprocess)
-        y_train = dataset[Const.label_col]
+    def fit(self, dataset: pd.DataFrame, labels: Optional[pd.Series]) -> None:
 
         self.pipeline = ImbPipeline(
             [
-                (
-                    "tfidf",
-                    TfidfVectorizer(
-                        min_df=2,
-                        max_df=0.5,
-                        stop_words=self.spacy_stopwords,
-                        max_features=100000,
-                        ngram_range=(1, 3),
-                    ),
-                ),
                 # ('feature_selection', SelectPercentile(chi2, percentile=50)),
                 ("sampling", RandomOverSampler()),
                 ("clf", self.config.classifier),
             ],
             verbose=True,
         )
 
-        self.pipeline.fit(X_train, y_train)
+        self.pipeline.fit(dataset, labels)
 
     def predict(self, dataset: pd.DataFrame) -> pd.DataFrame:
-        dataset = dataset[Const.input_col].swifter.apply(self.preprocess)
+        # dataset = dataset[Const.input_col].swifter.apply(self.preprocess)
```

Comment:
Should we keep these in commented out?

Reply:
no, I'll remove it, thanks for spotting!